scripts/clean_project.py

```
@@ -1,158 +1,80 @@
-            print(f"   - {rec}")
+        print(f"\n📊 Statistics:")
+        for category, count in result["stats"].items():
+            print(f"   - {category}: {count}")
 
-        return response.payload
+        return result
     else:
-        print(f"❌ Analysis failed: {response.payload.get('error', 'Unknown error')}")
-        return None
+        print("✅ No issues found!")
+        return result
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)